drivers/ieee802154/ieee802154_b9x.c

```diff
@@ -1051,9 +1055,11 @@ static int b9x_stop(const struct device *dev)
 		riscv_plic_irq_disable(DT_INST_IRQN(0) - CONFIG_2ND_LVL_ISR_TBL_OFFSET);
 		rf_set_tx_rx_off();
 #ifdef CONFIG_PM_DEVICE
+#if  CONFIG_SOC_RISCV_TELINK_B91 || CONFIG_SOC_RISCV_TELINK_B92
 		rf_baseband_reset();
-#if CONFIG_SOC_RISCV_TELINK_B91 || CONFIG_SOC_RISCV_TELINK_B92
 		rf_reset_dma();
+#elif CONFIG_SOC_RISCV_TELINK_B95
+		rf_radio_reset();
```

Comment:
just as improvement we can create "rf_radio_reset" function for b91 and b92 where put rf_baseband_reset(); rf_reset_dma(); to avoid additional #if

Reply:
I think it's for pm setting , we have not proc yet , it's need to reserve for B95 pm proc later .

Reply:
ok